src/Workspaces/Core/Portable/FindSymbols/SyntaxTree/SyntaxTreeIndex_Persistence.cs

```diff
@@ -136,14 +136,15 @@ public void WriteTo(ObjectWriter writer)
             var contextInfo = ContextInfo.TryReadFrom(reader);
             var declarationInfo = DeclarationInfo.TryReadFrom(stringTable, reader);
             var extensionMethodInfo = ExtensionMethodInfo.TryReadFrom(reader);
+            var globalAttributeInfo = GlobalAttributeInfo.TryReadFrom(reader);
```

Comment:
you also need to update the persistence version on line 18.  or else we'll throw exceptions trying to read existing persisted data into the new format.

Reply:
Thanks!